sprocs/grading_jobs_insert_internal.sql

```diff
@@ -69,6 +77,18 @@ BEGIN
     WHERE
         s.id = submission_id;
 
+    IF new_gradable = FALSE THEN
+        -- If the submission is ungradable then we should stop here
+
+        IF assessment_instance_id IS NOT NULL THEN
+            UPDATE instance_questions
+            SET status = 'invalid'::enum_instance_question_status
+            WHERE id = instance_question_id;
+        END IF;
+
+        RETURN;
+    END IF;
+
     -- ######################################################################
     -- update the variant
 
```

Comment:
Should we also run the `variant` update in the case that `gradable=false`? I guess so.

The `grading_job` that we insert below seems to have `graded_at=NULL`? But in the assessment log we explicitly try and find grading jobs with `graded_at IS NOT NULL` for internal grading jobs: https://github.com/PrairieLearn/PrairieLearn/blob/master/pages/instructorAssessmentInstance/instructorAssessmentInstance.sql#L248

I think these entries are just not showing up in the log, which is actually a bug. To fix this, we should insert the new grading job with `graded_at=now()`. I think we should do this even in the case that `gradable=false`.